### PR TITLE
Fixes #1606, add tests for webjars and make them work correctly

### LINKED ISSUE #1606: Test webjar support
We should test if we currently support webjars (in JVM mode and native).

A good test for that might be to deploy the Swagger-UI webjar in an application.

@geoand wondering if you might be interested in this one?

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Review by gsmet — COMMENTED** [review]
I added a few questions.

File: devtools/maven/src/test/resources/projects/classic/pom.xml [hunk 1]
```diff
@@ -27,6 +27,11 @@
       <artifactId>quarkus-undertow-websockets</artifactId>
       <version>${quarkus.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.webjars</groupId>
+      <artifactId>bootstrap</artifactId>
+      <version>3.1.0</version>
+    </dependency>
```
[the remark this answers is not in the document]
Comment (stuartwdouglas) [reply devtools/maven/src/test/resources/projects/classic/pom.xml]: We need to test that it works in both dev mode, native mode and JVM mode

File: bom/pom.xml [hunk 2]
```diff
@@ -150,6 +150,8 @@
         <jetty.version>9.4.15.v20190215</jetty.version>
         <flyway.version>5.2.4</flyway.version>
         <yasson.version>1.0.3</yasson.version>
+        <!-- Used for integration tests, to make sure webjars work-->
+        <bootstrap.version>3.1.0</bootstrap.version>
```
[the remark this answers is not in the document]
Comment (stuartwdouglas) [reply bom/pom.xml]: I don't think it really matters, even if they go out of sync there is no harm in testing two different versions.

**Review by gsmet — APPROVED** [review]
Looks good to me.

Needs a rebase though as `ServletTestCase` has been moved around.

**Comment by stuartwdouglas** [comment]
This was merged as part of the hot deployment multi module work

**Pull request closed without merging**